src/rplidar_node.cpp

```diff
@@ -112,7 +112,8 @@ rplidar_node::rplidar_node(const rclcpp::NodeOptions & options)
   /* done setting up RPLIDAR stuff, now set up ROS 2 stuff */
 
   /* create the publisher for "/scan" */
-  m_publisher = this->create_publisher<LaserScan>(topic_name_, 10);
+  rclcpp::QoS scan_qos(rclcpp::KeepLast(5),rmw_qos_profile_sensor_data);
+  m_publisher = this->create_publisher<LaserScan>(topic_name_, scan_qos);
```

Comment:
I'm not sure about making this the default. This would break existing behavior (as subscriptions would also need to be updated).

Could we add this as a parameter instead?